.github/workflows/rust-publish-dry-run-v2.yml

```diff
@@ -37,6 +37,12 @@ jobs:
       CARGO_TARGET_AARCH64_UNKNOWN_LINUX_GNU_LINKER: aarch64-linux-gnu-gcc
       RUST_BACKTRACE: 1
     steps:
+    - name: Remove unnecessary files
+      if: inputs.target != 'x86_64-pc-windows-msvc' && inputs.target != 'x86_64-pc-windows-msvc'
+      run: |
+        sudo rm -rf /usr/share/dotnet
+        sudo rm -rf /opt/hostedtoolcache
+
```

Comment:
Can I suggest we put this in a new action, `stellar/actions/disk-cleanup`, because maybe we would benefit from using this across all our disk resouce sensitive jobs, which is pretty much all our rust workflows.

Reply:
Done!